docs/rules/arrow-body-style.md

```diff
@@ -4,14 +4,17 @@ Arrow functions can omit braces when there is a single statement in the body. Th
 
 ## Rule Details
 
-This rule can enforce the use of braces around arrow function body.
+Arrow functions have two syntactic forms for their function bodies.  They may be defined with a *block* body (denoted by curly braces) `() => { ... }` or with a single expression `() => ...`, whose value is implicitly returned.
+
+This rule may be used to standardize your usage and thus avoid syntactic ambiguity.
```

Comment:
I think maybe " This rule can enforce or disallow the use of braces around arrow function body." is a more easily understandable way of saying this.